tensorflow_probability/python/experimental/mcmc/sequential_monte_carlo_kernel.py

```diff
@@ -49,6 +49,9 @@ class WeightedParticles(collections.namedtuple(
       `exp(reduce_logsumexp(log_weights, axis=0)) == 1.`. These must be used in
       conjunction with `particles` to compute expectations under the target
       distribution.
+    extra: a (structure of) Tensor(s) each of shape
+      `concat([[num_particles, b1, ..., bN], event_shape])`, where `event_shape`
```

Comment:
I think `num_particles` shouldn't be in the shape here.  (And could the comment mention that this represents global state of the sampling process that is *not* associated with individual particles?)